packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,105 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import icons from './icons/index.js';
+import Stack from './Stack.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: $Keys<typeof icons>,
+  iconAccessibilityLabel?: string,
+  accessibilityLabel: string,
+  accessibilityControls: string,
+  summary?: Array<string>,
+  isCollapsed: boolean,
+  onModuleClicked: boolean => void,
+  type: 'error' | 'info',
+  children?: Node,
+|};
+
+export default function ExpandableModuleBase({
+  title,
+  icon,
+  iconAccessibilityLabel,
+  accessibilityLabel,
+  accessibilityControls,
+  summary,
+  isCollapsed,
+  onModuleClicked,
+  type = 'info',
+  children,
+}: Props): Node {
+  const EXPANDABLE_TYPE_ATTRIBUTES = {
+    info: {
+      icon,
```

Comment:
move the title icon into 'info', so we will display either info icon or error icon.